src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByEmailController.cs

```diff
@@ -98,14 +98,14 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, SearchByEmailSubmi
     public async Task<IActionResult> EmailSearchInviteAlreadySent([FromRoute] int ukprn, CancellationToken cancellationToken)
     {
         var sessionModel = _sessionService.Get<AddEmployerSessionModel>();
-        var email = sessionModel?.Email;
+        var requestId = sessionModel?.RequestId;
```

Comment:
Can we use TempData instead of SessionService as there is no need to RequestId to persist in the session.

Reply:
SessionModel is not needed till we actually decide to take user to PAYE page

Reply:
will do